tools/test/wasm-js/wabt_patch

```diff
@@ -11,6 +11,23 @@ index 9b365e87..d770afa4 100644
                 memories.size());
    if (memories[memory].is_64) {
      return ReadU64Leb128(out_value, desc);
+diff --git src/common.h src/common.h
+index dbac0a89..1c0a57e8 100644
+--- src/common.h
++++ src/common.h
+@@ -66,11 +66,8 @@
+   char fixed_buf[WABT_DEFAULT_SNPRINTF_ALLOCA_BUFSIZE];                    \
+   char* buffer = fixed_buf;                                                \
+   size_t len = wabt_vsnprintf(fixed_buf, sizeof(fixed_buf), format, args); \
++  (void)len;                                                               \
```

Comment:
Is this line needed for remove warning?

Reply:
Yes because `len` is defined but nowhere used.
`len` could be removed too but it requires a lot of changes in source code, so I simply add meaningless statement here to avoid the unused variable warning.

Reply:
Ok. I see